collect_app/src/main/java/org/odk/collect/android/utilities/FileUtil.java

```diff
@@ -19,4 +24,36 @@ public String getRandomFilename() {
     public File getItemsetFile(String mediaFolderPath) {
         return new File(mediaFolderPath + "/itemsets.csv");
     }
+
+    public static String getFileContents(final File smsFile) throws IOException {
```

Comment:
You can shorten this significantly - https://stackoverflow.com/a/326440/137744

Reply:
Ah, ok, I see https://developer.android.com/reference/java/nio/file/Files.html#readAllLines(java.nio.file.Path,%20java.nio.charset.Charset) I always forget Android is not exactly Java.